usr/share/rear/init/default/001_verify_config_arrays.sh

```diff
@@ -0,0 +1,22 @@
+# ensure that all array variables defined in default.conf are still array variables after
+# reading the user configuration, fixes common user config error: https://github.com/rear/rear/issues/2930
+
+array_variables=($(
+    sed -n -e '/^[A-Z_]\+=(/s/=.*//p' "$SHARE_DIR"/conf/default.conf
+))
+
+
+for config in "$CONFIG_DIR"/{site,local,rescue}.conf "${CONFIG_APPEND_FILES_PATHS[@]}"; do
+    test -r "$config" || continue
+    for var in "${array_variables[@]}"; do
+        mapfile -t var_assignments < <(
+            sed -n -E -e "/$var\+?=/p" "$config"
+            )
+        for line in "${var_assignments[@]}"; do
+            [[ "$line" == *$var?(+)=\(* ]] || Error "Missing array assignment like +=(...) for $var in $config:$LF$line$LF"
```

Comment:
Better only tell what the actual error is
and avoid to teach the user what he should do.
So
```
... || Error "$var not assigned as array in $config: '$line'"
```
is simpler and more to the point.

Reply:
:heavy_check_mark: